app/templates/settings/imports.hbs

```diff
@@ -49,13 +49,13 @@
               <div id="import-tab-{{index}}" class="collapse" role="tabpanel">
                 <div class="card-block">
                   <p class="card-text">
-                    <span class="font-weight-bold">Status:</span> {{import.status}}<br>
+                    <span class="font-weight-bold">{{t "settings.imports.status"}}</span> {{import.status}}<br>
                     {{#unless (eq import.status "queued")}}
-                      <span class="font-weight-bold">Total Entries Found:</span> {{import.total}}<br>
+                      <span class="font-weight-bold">{{t "settings.imports.entries"}}</span> {{import.total}}<br>
                       {{#if (eq import.status "failed")}}
                         <br>
                         <small>
-                          It seems like your import has failed :( &mdash; We want to get this fixed for you so please <a href="mailto:help@kitsu.io">email</a> us.
+                          {{t "settings.imports.error-part1"}} <a href="mailto:help@kitsu.io">{{t "settings.imports.error-part2"}}</a>{{t "settings.imports.error-part3"}}
```

Comment:
This should be 1 translatable string. Use `{{t "settings.imports.error" htmlSafe=true}}`